src/app/item-popup/ItemPopupHeader.tsx

```diff
@@ -34,26 +35,30 @@ export default function ItemPopupHeader({
   noLink?: boolean;
 }) {
   const [showArmory, setShowArmory] = useState(false);
+  useHotkey('a', t('Hotkey.Armory'), () => setShowArmory(true));
 
   const showElementIcon = Boolean(item.element);
 
   const linkToArmory = !noLink && item.destinyVersion === 2;
 
   return (
-    <div
+    <button
+      type="button"
```

Comment:
I guess this button should be disabled when `!linkToArmory` to correctly reflect that nothing happens when clicking? Right now the cursor turns into a pointer even when `!linkToArmory`.